eZ/Publish/API/Repository/Tests/ContentTypeServiceAuthorizationTest.php

```diff
@@ -145,6 +145,62 @@ public function testDeleteContentTypeGroupThrowsUnauthorizedException()
     }
 
     /**
+     * Test for the createContentTypeGroup() method.
```

Comment:
Nitpick: the test is not really for `createContentTypeGroup()` method but `createContentType()`